package.json

```diff
@@ -20,8 +20,9 @@
     "test": "lerna run --concurrency 1 test",
     "test:ci": "lerna run --concurrency 1 test:ci",
     "check:lint": "lerna run check:lint",
-    "check:size": "lerna run check:size",
-    "check:size:build": "lerna run check:size:build",
+    "check:size": "lerna run check:size --scope \"@rudderstack/analytics-js\" --scope \"@rudderstack/analytics-js-plugins\"",
+    "check:size:build": "lerna run check:size:build --scope \"@rudderstack/analytics-js\" --scope \"@rudderstack/analytics-js-plugins\"",
+    "check:size:json": "lerna run check:size:json",
```

Comment:
why we need the --json output?

Reply:
This is for the size-limit action to forward the command to individual packages. Otherwise, it'd execute `npx size-limit --json` on the root directory and tries to compare it with the base for computing the size delta.

Reply:
Addressed.